src/components/SelectItemStatus/SelectItemStatus.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Select } from '@folio/stripes-components';
+import { FormattedMessage } from 'react-intl';
+import { ITEM_STATUS } from './const';
+import css from '../CheckIn/ItemsListModal.css';
```

Comment:
I don't think this is the right way to import styles. AFAIK, this is css-modules and css is tightly coupled to Component

Reply:
fixed